re.sonny.Workbench.Devel.json

```diff
@@ -224,7 +224,6 @@
         }
       ],
       "post-install": [
-        "cp -a /usr/share/doc/. /app/share/doc/",
```

Comment:
couldn't make this work so far

> cp: cannot overwrite directory '/app/share/doc/./appstream' with non-directory

Reply:
Maybe `--no-clobber` will allow it to copy without hitting the error? It might just bail though, not sure.

Reply:
nope,

I don't understand the problem. There is no `appstream` directory in `/app/share/doc/` and the same command in an interactive shell works fine